### PR TITLE
feat(module:skeleton): add nzRound prop and skeleton-element image

### PR DESCRIPTION
**Pull request by yangjunhan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

Aim to sync with the latest Ant-design skeleton component. Ant-design relevant issues:

- feat: Skeleton add round prop (https://github.com/ant-design/ant-design/pull/24137)

- feat: add Skeleton-Image (https://github.com/ant-design/ant-design/pull/24805)

## What is the new behavior?

Show paragraph and title radius when nzRound is true.

Add new nz-skeleton-element [nzType="image"].

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/skeleton/skeleton-element.component.ts [hunk 1]
```diff
@@ -86,10 +84,25 @@ export class NzSkeletonElementAvatarComponent implements OnChanges {
...
+    <span [class.ant-skeleton-image]="true">
+      <svg [class.ant-skeleton-image-svg]="true" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg">
+        <path [attr.d]="path" [class.ant-skeleton-image-path]="true" />
+      </svg>
+    </span>
+  `
+})
+export class NzSkeletonElementImageComponent {
+  path: string =
+    'M984.615385 196.923077c0-43.323077-35.446154-78.769231-78.769231-78.769231H118.153846c-43.323077 0-78.769231 35.446154-78.769231 78.769231v630.153846c0 43.323077 35.446154 78.769231 78.769231 78.769231h787.692308c43.323077 0 78.769231-35.446154 78.769231-78.769231V196.923077zM779.815385 748.307692h-571.076923c-23.630769 0-37.415385-25.6-25.6-45.292307l173.292307-301.292308c7.876923-13.784615 25.6-13.784615 33.476923 0l104.369231 179.2c7.876923 11.815385 25.6 13.784615 33.476923 1.969231l84.676923-122.092308c7.876923-11.815385 25.6-11.815385 33.476923 0L801.476923 708.923077c11.815385 17.723077 0 39.384615-21.661538 39.384615zM728.615385 393.846154c-43.323077 0-78.769231-35.446154-78.769231-78.769231s35.446154-78.769231 78.769231-78.769231 78.769231 35.446154 78.76923 78.769231-35.446154 78.769231-78.76923 78.769231z';
```
Comment (wenqi73) [inline components/skeleton/skeleton-element.component.ts:107]: 两边的图不一样，检查下

**Comment by wenqi73** [comment]
测试用例要加上

**Review by vthinkxie — CHANGES_REQUESTED** [review]
element 选项应当合并

File: components/skeleton/skeleton-element.component.ts [hunk 2]
```diff
@@ -86,10 +84,25 @@ export class NzSkeletonElementAvatarComponent implements OnChanges {
...
+      <svg [class.ant-skeleton-image-svg]="true" viewBox="0 0 1098 1024" xmlns="http://www.w3.org/2000/svg">
+        <path [attr.d]="path" [class.ant-skeleton-image-path]="true" />
+      </svg>
+    </span>
+  `
+})
+export class NzSkeletonElementImageComponent {
+  path: string =
+    'M365.714286 329.142857q0 45.714286-32.036571 77.677714t-77.677714 32.036571-77.677714-32.036571-32.036571-77.677714 32.036571-77.677714 77.677714-32.036571 77.677714 32.036571 32.036571 77.677714zM950.857143 548.571429l0 256-804.571429 0 0-109.714286 182.857143-182.857143 91.428571 91.428571 292.571429-292.571429zM1005.714286 146.285714l-914.285714 0q-7.460571 0-12.873143 5.412571t-5.412571 12.873143l0 694.857143q0 7.460571 5.412571 12.873143t12.873143 5.412571l914.285714 0q7.460571 0 12.873143-5.412571t5.412571-12.873143l0-694.857143q0-7.460571-5.412571-12.873143t-12.873143-5.412571zM1097.142857 164.571429l0 694.857143q0 37.741714-26.843429 64.585143t-64.585143 26.843429l-914.285714 0q-37.741714 0-64.585143-26.843429t-26.843429-64.585143l0-694.857143q0-37.741714 26.843429-64.585143t64.585143-26.843429l914.285714 0q37.741714 0 64.585143 26.843429t26.843429 64.585143z';
```
Comment (wenqi73) [inline components/skeleton/skeleton-element.component.ts:107]:
```suggestion
    <span class="ant-skeleton-image">
      <svg class="ant-skeleton-image-svg" viewBox="0 0 1098 1024" xmlns="http://www.w3.org/2000/svg">
        <path d="M365.714286 329.142857q0 45.714286-32.036571 77.677714t-77.677714 32.036571-77.677714-32.036571-32.036571-77.677714 32.036571-77.677714 77.677714-32.036571 77.677714 32.036571 32.036571 77.677714zM950.857143 548.571429l0 256-804.571429 0 0-109.714286 182.857143-182.857143 91.428571 91.428571 292.571429-292.571429zM1005.714286 146.285714l-914.285714 0q-7.460571 0-12.873143 5.412571t-5.412571 12.873143l0 694.857143q0 7.460571 5.412571 12.873143t12.873143 5.412571l914.285714 0q7.460571 0 12.873143-5.412571t5.412571-12.873143l0-694.857143q0-7.460571-5.412571-12.873143t-12.873143-5.412571zM1097.142857 164.571429l0 694.857143q0 37.741714-26.843429 64.585143t-64.585143 26.843429l-914.285714 0q-37.741714 0-64.585143-26.843429t-26.843429-64.585143l0-694.857143q0-37.741714 26.843429-64.585143t64.585143-26.843429l914.285714 0q37.741714 0 64.585143 26.843429t26.843429 64.585143z" class="ant-skeleton-image-path" />
      </svg>
    </span>
  `
})
export class NzSkeletonElementImageComponent {
```

**Review by wenqi73 — CHANGES_REQUESTED** [review]

**Pull request merged**